### PR TITLE
[unified sdk] Add MediaRenderer

### PR DESCRIPTION
**Pull request by MananTank**
## PR-Codex overview
This PR updates dependencies and adds support for the `@google/model-viewer` package. It also introduces new components and functions related to media rendering.

### Detailed summary
- Added `@google/model-viewer` package
- Added `MediaRenderer` component and related functions
- Updated `lit` dependency to version 2.8.0
- Introduced new icons for media rendering
- Updated mime type resolution functionality

> The following files were skipped due to too many changes: `pnpm-lock.yaml`, `packages/thirdweb/src/react/ui/MediaRenderer/MediaRenderer.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/thirdweb/package.json [hunk 1]
```diff
@@ -152,6 +152,7 @@
     "@coinbase/wallet-sdk": "3.7.2",
     "@emotion/react": "11.11.4",
     "@emotion/styled": "11.11.0",
+    "@google/model-viewer": "^2.1.1",
```
Comment (jnsdls) [inline packages/thirdweb/package.json:155]: @MananTank pretty sure there's a 3.0 of this - can we upgrade as part of the port?

also let's pin specific versions please

File: .changeset/lemon-carpets-argue.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"thirdweb": patch
+---
+
+Add MediaRenderer component
```
Comment (jnsdls) [inline .changeset/lemon-carpets-argue.md:5]: don't need a changeset here -> this will all be one big changeset for `beta` release later

**Review by jnsdls — COMMENTED** [review]
right now this is pretty much a straight port, can we improve it further?

File: packages/thirdweb/package.json (hunk 1 above)
Comment (MananTank) [reply packages/thirdweb/package.json]: done

File: .changeset/lemon-carpets-argue.md (hunk 2 above)
Comment (MananTank) [reply .changeset/lemon-carpets-argue.md]: removed

**Pull request merged**